templates/components/itilobject/actors/main.html.twig

```diff
@@ -233,6 +241,13 @@
       modal.find("input[name=_notifications_use_notification]").prop('checked', parseInt(actor.use_notification));
       modal.find("input[name=_notifications_alternative_email]").val(actor.alternative_email);
 
+      console.log(actor.use_notification);
+      if (parseInt(actor.use_notification) == false) {
```

Comment:
you may reduce this to (check the second parameter is correctly set, I'm not 100% sure):

```
modal.find("#alert_notification_disable").toggleClass('d-none', parseInt(actor.use_notification) == false);
```

Reply:
done